packs/data/equipment-effects.db/effect-oil-of-potency.json

```diff
@@ -0,0 +1,120 @@
+    "name": "Effect: Oil of Potency",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Oil of Potency]{Oil of Potency}</p>\n<p>If the item is a weapon, it temporarily becomes a +1 striking weapon, or, if it's armor, it temporarily becomes +1 resilient armor. This lasts for 1 minute.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 2
+        },
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "Armor",
+                        "value": "armor"
+                    },
+                    {
+                        "label": "Weapon",
+                        "value": "weapon"
+                    }
+                ],
+                "flag": "oilitemType",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.ArmorOrWeapon",
+                "rollOption": "potency",
+                "selection": "null"
+            },
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "flag": "weapon",
+                "key": "ChoiceSet",
+                "predicate": [
+                    "potency:weapon"
+                ],
+                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
+            },
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "types": [
+                        "armor"
+                    ]
+                },
+                "flag": "armor",
+                "key": "ChoiceSet",
+                "predicate": [
+                    "potency:armor"
+                ],
+                "prompt": "PF2E.SpecificRule.Prompt.Armor"
+            },
+            {
+                "key": "WeaponPotency",
+                "predicate": [
+                    "potency:weapon"
+                ],
+                "selector": "{item|flags.pf2e.rulesSelections.weapon}-attack",
+                "value": 1
+            },
+            {
+                "key": "Striking",
+                "predicate": [
+                    "potency:weapon"
+                ],
+                "selector": "{item|flags.pf2e.rulesSelections.weapon}-damage",
+                "value": 1
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "add",
+                "predicate": [
+                    "armor:id:{item|flags.pf2e.rulesSelections.armor}",
+                    "bonus:type:item"
+                ],
+                "selector": "ac",
+                "value": 1
+            },
```

Comment:
We may need to restrict this to armor without a potency rune already. A +1 armor is a valid target for this, and should still gain resilient but with no AC bump.

Reply:
I agree, and I tried really hard to make it happen but I could not figure it out. I think it might possible by adding something to the predicate. I figured that once I did the pr somebody more knowledgeable than me might point out how. I'll do some more digging though.